docs/src/chapters/testy.adoc

```diff
@@ -1,12 +1,262 @@
 [#_testy]
 == Testy
+Aby już na wczesnym etapie wykrywać powstałe błędy, zespół zdecydował o tworzeniu testów podczas pisania poszczególnych
+elementów systemu. Dzięki takiemu podejściu udało się uniknąć wielu błędów w trakcie rozbudowy aplikacji. Ułatwiło to
+również dokonywanie _code review_, ponieważ osoba sprawdzająca mogła w łatwy sposób zweryfikować czy dana funkcjonalność
+działa w taki sposób, jak to zaplanowano.
```

Comment:
"jak to zaplanowano" to nie jest błąd językowy?

Reply:
ok, trochę to zredaguję

Reply:
poprawiłem